src/design/Basics.js

```diff
@@ -135,6 +135,37 @@ var Basics = React.createClass({
     <span className="swatch graph-6"></span>
 
   </section>
+      
+  <section>
+    <a className="anchor" id="padding"></a>
+    <h2>Padding</h2>
+      
+      <p>Padding on elements and in mixins is based off of the inuitcss <code>$inuit-base-spacing-unit</code> variable, which is set to 24px. Hardcoded pixel values for padding should rarely be used; use spacing variables instead.</p>
+
+    <h3>Spacing Variables</h3>
```

Comment:
Our primary direction for consumers is that they should not have to do any CSS in their code. Also, this is a design document, not a development document. For this design document, I think we should document that Grommet aligns things to multiples of 24px, such as: 6px, 12px, 24px, 48px, 96px. The reason we want to communicate this is so that designers align their designs to these boundaries. For instance, all text paragraphs should have a 24px line height, spacing between objects should use these values, etc. The components in the Grommet sticker sheet should already reflect this.

Then, we should add some documentation about the Grommet components exposing sizing and padding via "small", "medium", and "large" sizes. For instance, the default Header is medium sized at 72px high. A small Header is 48px high. A large header is 96px high. Maybe we can explain the small/medium/large concept and then show where they are applied: Header, Box padding, Sidebar, Chart, Distribution, Meter, Icon, Paragraph, Tiles, Video.